lib/ansible/modules/monitoring/datadog_monitor.py

```diff
@@ -296,6 +302,7 @@ def install_monitor(module):
         "escalation_message": module.params['escalation_message'],
         "notify_audit": module.boolean(module.params['notify_audit']),
         "locked": module.boolean(module.params['locked']),
+        "evaluation_delay": module.params['evaluation_delay'],
```

Comment:
If `null` (the default) is passed it does this mean no delay, or does this need to be zero?

Reply:
Ya, if you want "no delay", you don't need to define `evaluation_delay` into .yml